tests/manual/test-procedure.rst

```diff
@@ -249,6 +249,96 @@ Code block 1::
     display.show(Image.PACMAN)
 
 
+Test Case: module.py file can be loaded by file picker and used in main.py
+''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+- Load by Load/Save > Add file the file ``emoji.py``
+- [ ] Confirm the file shows up in the files list with the same title.
+- Return to the editor and replace the current script with the following::
+
+    from microbit import *
+    from emoji import *
+
+    while True:
+        display.show(😃)
+        if accelerometer.was_gesture('shake'):
+            display.show(😡)
+            sleep(2000)
+        if button_a.was_pressed():
+            display.show(💖)
+            sleep(2000)
+        elif button_b.was_pressed():
+            display.show(🏠)
+            sleep(2000)
+        sleep(100)
+
+- [ ] Flash this file to the micro:bit and confirm that it behaves as expected,
+    showing emojis for the appropriate gestures and buttons.
```

Comment:
I think this line might need to have only 2 spaces to be formatted correctly:
![image](https://user-images.githubusercontent.com/29712657/61804058-27702b00-ae2b-11e9-9045-9e218dda593f.png)

Reply:
The same applies to the other tests cases. You can see the generated output on GitHub: https://github.com/bbcmicrobit/PythonEditor/blob/674f599335a39714a2d6d21e86e10a2f62c4294a/tests/manual/test-procedure.rst